src/common/version.h.in

```diff
@@ -13,6 +13,7 @@ namespace ccf
   //
 
   static constexpr auto ccf_version = "@CCF_VERSION@";
+  static constexpr auto quickjs_version = "@QUICKJS_VERSION@";
```

Comment:
Can we return this in `/node/version` endpoint too?

Reply:
We can, what about all other dependencies?

Reply:
We can also do the other dependencies, but it's arguably less important because they don't typically have a large surface to KV types unlike the JS runtime (with the obvious exception of the JSON parser, but then JSON is standardised, unlike QuickJS bytecode).

Reply:
Ok, also done.